cost-analyzer/templates/cost-analyzer-db-pvc-template.yaml

```diff
@@ -3,6 +3,7 @@
 {{- if not .Values.persistentVolume.dbExistingClaim -}}
 {{- if .Values.persistentVolume.enabled -}}
 {{- if .Values.persistentVolume.dbPVEnabled -}}
+{{- if and .Values.kubecostDeployment.statefulSet.enabled .Values.kubecostDeployment.leaderFollower.enabled -}}
```

Comment:
I don't think I would necessarily block users from installing this to just using a statefulset?

Reply:
good catch @AjayTripathy, before the changes proposed to be done for a better combination with `.Values.kubecostDeployment.leaderFollower.enabled` it was a negated `.Values.kubecostDeployment.statefulSet.enabled` so I added `(not .Values.kubecostDeployment.statefulSet.enabled)` instead